test/e2e/page-objects/pages/ErrorPage.ts

```diff
@@ -0,0 +1,78 @@
+import { strict as assert } from 'assert';
+import { Driver } from '../../webdriver/driver';
+import HeaderNavbar from './header-navbar';
+import SettingsPage from './settings-page';
+import DevelopOptions from './deveoper-options';
+
+class ErrorPage {
+  private readonly driver: Driver;
+
+  // Locators
+  private readonly errorPageTitle: object = {
+    text: 'MetaMask encountered an error',
+    css: 'h3',
+  };
+
+  private readonly errorMessage = '[data-testid="error-page-error-message"]';
+
+  private readonly sendReportToSentryButton =
+    '[data-testid="error-page-describe-what-happened-button"]';
+
+  private readonly sentryReportForm = '#sentry-feedback';
+
+  private readonly contactSupportButton =
+    '[data-testid="error-page-contact-support-button"]';
+
+  constructor(driver: Driver) {
+    this.driver = driver;
+  }
+
+  async check_pageIsLoaded(): Promise<void> {
+    try {
+      await this.driver.waitForSelector(this.errorPageTitle);
+    } catch (e) {
+      console.log('Timeout while waiting for Error page to be loaded', e);
+      throw e;
+    }
+    console.log('Error page is loaded');
+  }
+
+  async triggerPageCrash(): Promise<void> {
+    const headerNavbar = new HeaderNavbar(this.driver);
+    await headerNavbar.openSettingsPage();
+    const settingsPage = new SettingsPage(this.driver);
+    await settingsPage.check_pageIsLoaded();
+    await settingsPage.goToDevelopOptionSettings();
+
+    const developOptionsPage = new DevelopOptions(this.driver);
+    await developOptionsPage.check_pageIsLoaded();
+    await developOptionsPage.clickGenerateCrashButton();
+  }
+
+  async validate_errorMessage(): Promise<void> {
+    const errorMessageTextDOM = await this.driver.waitForSelector(
+      this.errorMessage,
+    );
+    const errorMessageText = await errorMessageTextDOM.getText();
+    assert.equal(
+      errorMessageText,
+      `Message: Unable to find value of key "developerOptions" for locale "en"`,
+      `Error in loading error page`,
+    );
+  }
+
+  async submitToSentryUserFeedbackForm(): Promise<void> {
+    console.log(`Open sentry user feedback form in error page`);
+    await this.driver.clickElement(this.sendReportToSentryButton);
+    await this.driver.waitForSelector(this.sentryReportForm);
+  }
+
+  async contactAndValidateMetamaskSupport(): Promise<void> {
+    console.log(`Contact metamask support form in a separate page`);
+    await this.driver.clickElement(this.contactSupportButton);
+    // metamask, help page
+    await this.driver.waitUntilXWindowHandles(2);
```

Comment:
I'm a bit unsure about this line :thinking: I'm thinking in the case of tests which have more windows open.

If a new page is open, maybe we could look for that page title (`'Hello from MetaMask Help Center 🦊♥️ | MetaMask Help Center 🦊♥️'`) or for the expected url (`https://support.metamask.io/`) instead of the number of windows (which could change depending on the test).
Alternatively, we could wait for previous number of windows +1.

Reply:
maybe we can still avoid switching to the new page (which can avoid us problems with live requests)

Reply:
the original implementation was when we click the button it brings us to a new page, i was hesitating to use an arbitrary title, as if it changes we'll need to modify our test, which is relying on 3rd party. I'll use `await this.driver.waitUntilXWindowHandles(1)` before the click to verify. => [ff4748](https://github.com/MetaMask/metamask-extension/pull/27857/commits/ff4748e3c34ac7f17c55686d0756182c8791ccc3)

>maybe we can still avoid switching to the new page (which can avoid us problems with live requests)

This seems like a good idea, can you show me an example to prevent this `window.open(SUPPORT_REQUEST_LINK, '_blank');` event? thank would be very helpful